docs/content/concepts/assets/asset-checks/define-execute-asset-checks.mdx

```diff
@@ -237,6 +237,41 @@ defs = Definitions(
 
 There are a variety of types supported via the <PyObject object="MetadataValue" /> class. You can view the metadata on the **Checks** tab of the **Asset details** page.
 
+### Blocking downstream assets
+
+To block downstream assets from executing when checks fail, set the `blocking` argument to `True` in the <PyObject object="asset_check" /> decorator. In the following example, `check_upstream_asset` will block `downstream_asset` from executing.
+
+```python file=/concepts/assets/asset_checks/blocking.py
+from dagster import AssetCheckResult, Definitions, asset, asset_check
+
+
+@asset
+def upstream_asset():
+    pass
+
+
+@asset_check(asset=upstream_asset, blocking=True)
+def check_upstream_asset():
+    return AssetCheckResult(passed=False)
+
+
+@asset(deps=[upstream_asset])
+def downstream_asset():
+    pass
+
+
+defs = Definitions(
+    assets=[upstream_asset, downstream_asset], asset_checks=[check_upstream_asset]
+)
+```
+
+When the `blocking` toggle is enabled, downstream assets will wait to execute until the check completes. If the check returns a failing <PyObject object="AssetCheckResult" /> or raises an exception, downstream assets will not execute. If the <PyObject object="AssetCheckSeverity"/> is set to `WARN` instead of `ERROR`, the downstreams will still execute.
```

Comment:
```suggestion
When `blocking` is enabled, downstream assets will wait to execute until the check completes:

- **Downstream assets will not execute** if the check returns a failing <PyObject object="AssetCheckResult" /> or raises an exception
- **Downstream assets will execute** if the <PyObject object="AssetCheckSeverity"/> is set to `WARN` instead of `ERROR`
```

Nit - this isn't a toggle

Also suggesting a list and moving the end result to the beginning of the sentence to make it easier to skim